src/components/Alerts/tsconfig.json

```diff
@@ -0,0 +1,12 @@
+{
+  "extends": "../../../tsconfig.build.json",
+  "compilerOptions": {
+    "rootDir": "src",
+    "outDir": "dist",
+    "tsBuildInfoFile": "tsconfig.tsbuildinfo",
+    "composite": true
+  },
+  "include": ["src/**/*"],
+  "exclude": ["dist", "src/*.stories.tsx"],
```

Comment:
I think it's better to exclude stories the following way: `src/stories/**/*`. Also it's not a `.tsx`